assets/dev/js/editor/regions/panel/footer.js

```diff
@@ -134,7 +134,7 @@ module.exports = Marionette.ItemView.extend( {
 	},
 
 	onHistoryClick() {
-		$e.route( 'panel/history/actions' );
+		$e.route( 'panel/history/actions-v2' );
```

Comment:
in the future, we'll probably want to keep the original API (?), so maybe we should keep all those "old" API calls, hook into the route system and replace it to v2 conditionally (based on experiment or whatever)

WDYT?

Reply:
Done